app/services/budget/export.rb

```diff
@@ -52,10 +55,17 @@ def activity_data(activity)
         activity.partner_organisation_identifier,
         activity.extending_organisation&.name,
         I18n.t("table.body.activity.level.#{activity.level}"),
-        activity.title
+        activity.title,
+        activity.programme? ? activity_comments(activity) : ""
       ]
     end
 
+    def activity_comments(activity)
+      skip_bullet do
+        activity.comments.map { |comment| comment.body.strip }.join("|")
```

Comment:
If we handle this in a similar way to how we do in `app/models/export/activity_comments_column.rb`(using`:pluck`) we don't need to skip bullet here. 

So we could do `activity.comments.pluck(:body).map(&:strip).join("|")`

Although we're `:pluck`ing then `map`ping, it appears to be the tiniest bit faster on average after running the tests a few times, but that could just be luck.